docbrown/db/benches/common/mod.rs

```diff
@@ -104,6 +104,52 @@ pub fn run_ingestion_benchmarks<F>(
     );
 }
 
+fn times(n: usize) -> impl Iterator {
+    std::iter::repeat(()).take(n)
+}
+
+pub fn run_large_ingestion_benchmarks<F>(
+    group: &mut BenchmarkGroup<WallTime>,
+    mut make_graph: F,
+) where
+    F: FnMut() -> GraphDB,
+{
+    bench (
+        group,
+        "100k updates as new vertices",
+        Option::None,
+        |b: &mut Bencher| {
+            b.iter_batched_ref(
+                || (make_graph(), make_index_gen()),
+                |(g, index_gen)|
+                    for _ in times(100000) {
+                        g.add_vertex(index_gen.next().unwrap(), 0, &vec![])
```

Comment:
These two bench tests should perform the same action (add_vertex, add_edge or both), just on different graphs so that we can see if there is any difference in performance between a newly initialised graph and a graph with existing objects